src/templates_front/templator_front.org

```diff
@@ -3857,7 +3887,7 @@ def has_$group_str$(trexio_file) -> bool:
 
   #+begin_src f90 :tangle helper_fortran.f90
 contains
-   integer(8) function trexio_open (filename, mode, backend, rc_open)
+   integer(c_int64_t) function trexio_open (filename, mode, backend, rc_open)
```

Comment:
Should not this return `trexio_filetype` as declared at the beginning of `trexio_f.f90` ?

Reply:
It would be cleaner, indeed. And it should be done in all the functions. I push a new commit.